content/blog/2016/2016-08-16-jenkins-world-speaker-blog-aquient.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Jenkins World Speaker Highlight: Continuously Delivering Continuous Delivery Pipelines"
```

Comment:
This title is too long, we already published the one for Ben Patterson without the prefix "Jenkins World Speaker Highlight" so my preference would be just omit these in the future.